scripts/build_images/build_multiplatform.py

```diff
@@ -10,8 +11,6 @@
     print_and_run_command,
 )
 
-from .build import build_images
```

Comment:
This was broken because `pip install .` would generate an empty `build` folder. I updated `make python-deps-core` to handle this.